cmd/fyne/internal/commands/package-windows.go

```diff
@@ -63,6 +63,7 @@ func (p *Packager) packageWindows(tags []string) error {
 			CombinedVersion: p.combinedVersion(),
 		}
 		err := templates.ManifestWindows.Execute(manifestFile, tplData)
+		manifestFile.Close()
```

Comment:
Can you please do the close after the error? It is better to handle the error where it is created.